### DISCUSSION TITLE
Is it possible to customize the “prev” and “next” buttons?

### DISCUSSION THREAD

**Discussion by jorikfon** [retypeapp/retype#235] [Q&A]
Hello there. I read all your website and did not find the answer ;) 
We will use your software to make documentation for Russian customers, and it would be amazing to have a translation for every English word on the site. 
I found 🔍 only how to change the search placeholder... 
But how can I translate some buttons?

**Comment by geoffreymcgill** [comment]
With the current release, Retype does not have an option to translate the strings, but a future release certainly will. 

We are close to getting this functionality built. There were a few big pieces that needed to be finished first.

Once the functionality is in place, you should only have to rebuild your Retype project, then everything will be in Russian. Retype should be able to auto-detect the content language and then localize all the words (strings) within the app to that language. Of course, there will be an option to manually configure the website language as well as options to customize all strings.

Hope this helps.

## See also

- #18 
- #24

**Reply by geoffreymcgill** [reply]
The upcoming release of Retype v3.0 will include targeting a language, including Russian. Currently 24 languages are supported.

See https://github.com/retypeapp/retype/issues/404#issuecomment-1469152692

Hope this helps.
